src/main/java/com/contentful/java/cma/CMACallback.java

```diff
@@ -16,22 +16,20 @@
 
 package com.contentful.java.cma;
 
-import retrofit.RetrofitError;
-
 /**
  * Callback to be used with any of the {@code CMAClient} asynchronous methods.
- *
+ * <p>
```

Comment:
True, that was the autoformat. I'll remove this line and check the others if a selfclosing will be sufficient.